features/no_quench.go

```diff
@@ -0,0 +1,5 @@
+//go:build !quench
+
+package features
+
+const QuenchEnabled = false
```

Comment:
[small]there is already a file to create the VPN instance, wouldn't it make sense to combine this with that?

Reply:
Wouldn't be usable outside of the `cmd/daemon` in that case.